src/cli-util.js

```diff
@@ -10,12 +10,81 @@
  * governing permissions and limitations under the License.
  */
 
+const readline = require('readline');
+const chalk = require('chalk');
 const packagejson = require('../package.json');
 
 function getVersion() {
   return packagejson.version;
 }
 
+const spinnerFrames = process.platform === 'win32'
```

Comment:
Is this really a platform dependency or a terminal dependency, i.e. we can assume UTF-8 support when `$TERM=xterm`?

Reply:
don't know :-) I copy pasted it from somewhere. but let me try it out....

Reply:
in command prompt, auth doesn't work at all:

```
C:\Users\tripod>hlx --version
0.13.6-pre.9

C:\Users\tripod>hlx auth
Cannot read property 'split' of null
```

in mingw, it works. even with the unicode.

Reply:
apparently this is a known problem. https://github.com/sindresorhus/figures
I'd stick to the distinction for know.